doc/api/schemas/CodeRepositorySettings.yaml

```diff
@@ -0,0 +1,40 @@
+type: object
+description: Code Repository Settings
+properties:
+    projectUrl:
+        type: string
+    environment:
+        type: string
+    deploymentModes:
+        type: array
+        items:
+            $ref: './DeploymentMode.yaml'
```

Comment:
I believe deployment should be an attribute of a model, as it is a model that is deployed. One CodeRepo can be the source of many trained models with different deployments

Reply:
and what is wrong here? Do not understand.

Reply:
These are the `CodeRepositorySettings`. We want to deploy a model, not a code repository.